samtranslator/region_configuration.py

```diff
@@ -22,3 +24,9 @@ def is_apigw_edge_configuration_supported(cls):
             "aws-iso-b",
             "aws-cn",
         ]
+
+    @classmethod
+    def is_sar_supported(cls):
+        return boto3.Session().region_name not in [
```

Comment:
Can we have a docstring containing the URL in case `af-south-1` supports it we can quickly find the reference?

Reply:
Added